content/docs/for-buildpack-authors/concepts/layer.md

```diff
@@ -3,8 +3,12 @@ title="What is a buildpack dependency layer?"
 weight=99
 +++
 
+`Dependency layers` are semantically meaningful layers that are contributed by one or more buildpacks during the building phase, one for each dependency.
```

Comment:
We should link to https://buildpacks.io/docs/for-buildpack-authors/how-to/write-buildpacks/create-layer/ somewhere on this page (and vice versa)

Reply:
linked at the bottom as a note